Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -1066,7 +1066,7 @@ private void LoadTemplates(string path)
         public string GetFullPath(string filePath)
         {
             // if the path is root of a drive in windows, which normally returns C:, return C:\ instead.
-            if (filePath.Length==2 && filePath[0].ToString().ToUpper()[0] >='A' && filePath[0].ToString().ToUpper()[0] <='Z' && Path.DirectorySeparatorChar=='\\' && filePath[filePath.Length - 1] == ':')
+            if (filePath.Length==2 && filePath[0] !='/' && Path.DirectorySeparatorChar=='\\' && filePath[filePath.Length - 1] == ':')
```

Comment:
Fix formating and there is no reason to check `Path.DirectorySeparatorChar`.

Reply:
Wait, I'll just write you the line myself, it will be easier, need to wash my hands.

Reply:
```c#
if (filePath.Length == 2 && filePath[0] != '/')
```

Reply:
It may be possible for filePath to be a relative path (see line 1076). To prevent a match for "\a" and "aa", I check the second char for ":"in the last commit.

Reply:
Good point, move the code under `if (Path.IsPathRooted(filePath))`, forgot to mention it before.

Reply:
The problem with moving it under IsPathRooted is the case of the Pipleline executable in the root of the drive would fail to open a project.

By leaving it where it is and adding the colon check, there are no cases excepted and has minimal overhead.

Reply:
> The problem with moving it under IsPathRooted is the case of the Pipleline executable in the root of the drive would fail to open a project.

Why would it fail?

Reply:
Tho do note that I'm also talking about modifying the methods `if (project == null)` thingy as well, the build bot is down so once its back online I'll just fix this properly.

Reply:
https://pastebin.com/aQwbEm3B

Reply:
> Why would it fail?
The `_project==null` does not guarantee an absolute path.

Reply:
It does actually, if the project is not open, relative paths can't be resolved.

Reply:
Windows doesn't seem to care about the number of backslashes beyond the beginning of the path, which it interprets multiples as a UNC path.
We should be good to go with that last commit.

Thank you for working with me on this.